src/content/docs/en/guides/cms/gitcms.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: GitCMS & Astro
+description: Integrate GitCMS into your Astro project for seamless content management
+sidebar:
+  label: GitCMS
+type: cms
+service: GitCMS
+i18nReady: true
+---
+
+[GitCMS](https://gitcms.blog) is a Chrome extension that bridges the gap in static site generation (SSG) workflows by enhancing the writing and management experience for markdown files. It transforms GitHub into a powerful headless CMS, providing a Notion-like editing experience directly within GitHub's interface. With GitCMS, developers, clients, and content writers can write, manage, and search markdown content effortlessly — eliminating the pain of editing raw markdown or juggling file management. This makes managing content for Astro sites smoother, without additional servers or complex CMS setups.
```

Comment:
```suggestion
[GitCMS](https://gitcms.blog) is a Chrome extension that enhances GitHub's Markdown editing interface, creating a Git-based, headless CMS in your browser.
```
These introductions are always short, and factual. Not marketing-speak. Something like this is what we're going for. I believe this seems accurate based on what I researched, but you can suggest an alternative!